docs/src/main/asciidoc/_elytron/components/Distributed_Security_Realm.adoc

```diff
@@ -0,0 +1,34 @@
+[[distributed-realm]]
+= Distributed Realm
+
+The distributed realm allows combining identities located across multiple security realms of any type into a single security realm.
+
+The `distributed-realm` resource contains the following attributes: -
+
+* `realms` - List of realms in the order they should be queried
+* `ignore-unavailable-realms` - Whether subsequent realms should be checked after an unavailable realm is reached. The default value is false.
+* `emit-events` - Whether a SecurityEvent signifying realm unavailability should be emitted. The default value is true.
+
+== Defining a Distributed Realm
+
+As an example, assuming a `properties-realm`, `ldap-realm` and `jdbc-realm` have already been configured, then a distributed-realm that references these three realms can be created as follows:
+
+[source,options="nowrap"]
+----
+/subsystem=elytron/distributed-realm=distributedRealmExample:add(realms=[properties-realm, ldap-realm, jdbc-realm])
+----
+
+When configuring a `distributed-realm`, it is possible to ignore any unavailable realms by setting `ignore-unavailable-realms` to true. When retrieving an identity from a `distributed-realm`, each of the configured realms is invoked sequentially until a realm containing the identity is found. If a security realm becomes unavailable for some reason, if `ignore-unavailable-realms` is true, any subsequent realms will still be checked.
```

Comment:
It should be also clear what happens when `ignore-unavailable-realms=false` (default behavior).

Reply:
Hi @OndrejKotek , thanks, should be updated now.

Reply:
I suggest to be more descriptive and document that an exception is thrown in such case.

Reply:
@OndrejKotek updated